azurerm/internal/services/cosmos/resource_arm_cosmosdb_gremlin_graph.go

```diff
@@ -386,14 +388,18 @@ func resourceArmCosmosDbGremlinGraphRead(d *schema.ResourceData, meta interface{
 			d.Set("throughput", nil)
 		}
 	} else {
-		d.Set("throughput", throughputResp.Throughput)
+		if props := throughputResp.ThroughputSettingsGetProperties; props != nil {
```

Comment:
Since this is used in almost every resource I am wondering if we should pull this out to a helper function? Thoughts would be appreciated

Reply:
I could see the benefit of a helper here, returning a default value for this field 👍